config/webpack.dev.config.js

```diff
@@ -0,0 +1,28 @@
+const config = require('@redhat-cloud-services/frontend-components-config');
+const { setHtmlPlugin, setReplacePlugin, setCommonPlugins } = require('./build.plugins');
+const { setupDotenvFilesForEnv } = require('./build.dotenv');
+
+const { _BUILD_RELATIVE_DIRNAME, DEV_BRANCH, DEV_PORT } = setupDotenvFilesForEnv({
+  env: 'development'
+});
+
+const { config: webpackConfig, plugins } = config({
+  appUrl: ['/insights/subscriptions', '/openshift/subscriptions'],
+  debug: true,
+  deployment: 'apps',
+  env: (/(prod|qa|ci)(-stable|-beta)/.test(DEV_BRANCH) && DEV_BRANCH) || 'prod-stable',
+  port: Number.parseInt(DEV_PORT, 10),
+  rootFolder: _BUILD_RELATIVE_DIRNAME,
+  skipChrome2: true,
```

Comment:
@Hyperkid123 we're not clear why we have to disable this flag in order to get everything running?

Reply:
Yeah, we also need to modify the cloud services config in order to tell chrome that the application is ready. https://github.com/RedHatInsights/cloud-services-config/blob/ci-beta/docs/chrome/docs.md#note-configuration-bellow-applies-to-applications-not-using-chrome-2-rendering

I think it would be better to go ahead with this solution first to makes webpack 5 is doing what it's supposed to.